net/libfab/libfab.c

```diff
@@ -641,18 +671,21 @@ static int libfab_ep_find(struct m0_net_transfer_mc *tm, const char *name,
 			  struct m0_fab__ep_name *epn,
 			  struct m0_net_end_point **epp)
 {
-	struct m0_net_end_point *net;
-	struct m0_fab__ep       *xep;
-	char                     ep[LIBFAB_ADDR_STRLEN_MAX] = {0};
-	bool                     found = false;
-	int                      rc = 0;
+	struct m0_net_end_point  *net;
+	struct m0_fab__ep        *ep;
+	struct m0_fab__active_ep *aep;
+	struct m0_fab__tm        *ma;
+	char                      ep_str[LIBFAB_ADDR_STRLEN_MAX] = {0};
+	char                     *wc = NULL;
+	bool                      found = false;
+	int                       rc = 0;
 
 	M0_ENTRY();
 
 	m0_tl_for(m0_nep, &tm->ntm_end_points, net) {
-		xep = libfab_ep_net(net);
-		if (libfab_ep_cmp(xep, name, epn) == true) {
-			*epp = &xep->fep_nep;
+		ep = libfab_ep_net(net);
+		if (libfab_ep_cmp(ep, name, epn) == true) {
```

Comment:
According to "doc/coding-style",
```
137   * use difference between NULL, 0 and "false" to emphasize whether
138     an expression is used as a pointer, integer (including success
139     or failure code) or boolean:
140 
141           if (p == NULL) { /* assumes that p is a pointer */
142           } else if (q != 0) { /* q is an integer */
143           } else if (r) { /* r is a boolean */
144           }
145 
146       Specifically, never use if (r == true);

```
this line should be:
```
if (libfab_ep_cmp(ep, name, epn)) {
```

Reply:
Done